cli/args/flags.rs

```diff
@@ -2459,6 +2460,14 @@ Directory arguments are expanded to all contained files matching the glob
         .conflicts_with("inspect-brk")
         .help("Collect coverage profile data into DIR. If DIR is not specified, it uses 'coverage/'."),
     )
+    .arg(
+      Arg::new("clean")
+        .long("clean")
+        .help("Empty the coverage profile data directory before running tests.
+        
+Note: running multiple `deno test --clean` calls in series or parallel for the same coverage directory may cause race conditions.")
+        .action(ArgAction::SetTrue),
+    )
```

Comment:
This makes sense, but I'm wondering if we should reword this to say 
```
"Empty any temporary files created by `deno test` before running the tests. Currently this includes coverage data, but may change in the future."
```
Thoughts?

Reply:
I feel like saying it might change in the future might cause some confusion. IMO, how it is now is fine. At most, we could do "Empty the **temporary** coverage profile data..."

Reply:
Okay, that seems reasonable, please update the message